docs/framework-introscope_agent.md

```diff
@@ -1,5 +1,5 @@
-# Introscope Agent Framework
-The Introscope Agent Framework causes an application to be automatically configured to work with a bound [Introscope service][].  **Note:** This framework is disabled by default.
+# CA APM Framework
```

Comment:
If you want to change the branding, you'll need to change Ruby class, files, documentation, etc. to match.  We use conventions for rendering the names in the buildpack so things need to line up for consistency.  If you want to make the change, you should; users shouldn't have modified those files in anyway except possibly enabling the framework in `components.yml`.  Your change would result in the same net effect, and enabled CA APM Agent Framework.